Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,115 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponenet_EntityFallsUnderGravity)
+    {
+        PhysX::CharacterControllerComponent* characterComponent = nullptr;
+        PhysX::CharacterGameplayComponent* gameplayComponent = nullptr;
+        AzFramework::TransformComponent* transform = nullptr;
+
+        float expectedGravity = 1.5f;
+
+        // Create character
+        auto gameplayEntity = AZStd::make_unique<AZ::Entity>("GameplayEntity");
+        {
+            auto characterConfiguration = AZStd::make_unique<Physics::CharacterConfiguration>();
+            auto characterShapeConfiguration = AZStd::make_unique<Physics::CapsuleShapeConfiguration>();
+            auto characterGameplayConfiguration = AZStd::make_unique<PhysX::CharacterGameplayConfiguration>();
+
+            characterShapeConfiguration->m_height = 1.5f;
+            characterShapeConfiguration->m_radius = 0.5f;
+
+            characterGameplayConfiguration->m_gravityMultiplier = expectedGravity;
+            characterGameplayConfiguration->m_groundDetectionBoxHeight = 0.05f;
+
+            transform = gameplayEntity->CreateComponent<AzFramework::TransformComponent>();
+            characterComponent = gameplayEntity->CreateComponent<CharacterControllerComponent>(
+                AZStd::move(characterConfiguration), AZStd::move(characterShapeConfiguration));
+            gameplayComponent = gameplayEntity->CreateComponent<CharacterGameplayComponent>(AZStd::move(characterGameplayConfiguration));
+
+            transform->SetWorldTM(AZ::Transform::Identity());
+        }
+        gameplayEntity->Init();
+        gameplayEntity->Activate();
+
+        // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component
+        auto startTransform = transform->GetWorldTM();
+
+        int duration = 10;
+        float totalTime = 0.0f;
+        float timeStep = AzPhysics::SystemConfiguration::DefaultFixedTimestep;
+
+        if (auto* physXSystem = GetPhysXSystem())
+        {
+            for (int i = 0; i < duration; i++)
+            {
+                physXSystem->Simulate(timeStep);
+                totalTime += timeStep;
+            }
+        }
+
+        AZ::Transform endTransform = transform->GetWorldTM();            
+
+        EXPECT_THAT(endTransform, testing::Not(UnitTest::IsClose(startTransform)));
```

Comment:
I'd prefer to check for something more specific than just not being close - this could mean it was 1cm away from where it started or 1km. Should be possible to figure out how far it should have fallen in the number of timesteps that have passed, and you can always use a fairly big tolerance.

Reply:
@greerdv 

Trying to figure out how to calculate the gravity here.

Currently stuck on trying to figure out how to use transforms.

Is there an easy way to get the X,Y,Z positions or a Vector3?

Reply:
@greerdv I've added a gravity calculation in, but now I'm getting failures when testing against my end Position and my expected position.

I'm going to have to dig deeper into this unless you can see anything that I'm doing incorrectly here.

Reply:
Extracting this out into the backlog on internal tracking too. I will create GHIs when work resumes on this after beginning of the year planning finishes.